requirements.txt

```
@@ -1,8 +1,9 @@
+amazon-braket-pennylane-plugin
```

Wasn't sure if this should be pinned?
```suggestion
amazon-braket-pennylane-plugin==1.0.0
```